Exit if an error occurs while building.
This fix is already in icmake 10.00.00.
--- a/icmake/icm_bootstrap.orig	2024-09-30 02:27:52.000000000 -0500
+++ b/icmake/icm_bootstrap	2024-09-30 02:28:58.000000000 -0500
@@ -28,7 +28,7 @@
 for target in rss icmake un pp comp exec dep icmbuild
 do
     cd $target
-    ./icm_bootstrap
+    ./icm_bootstrap || exit 1
     cd ..
 done
 
